musicApp/R/functions.R

```diff
@@ -126,7 +137,6 @@ getSongLyrics = function(songname, artist)
 }
 
 
-####################################################################################################
 # function that returns vector containing words used in an album based on the name of the album and artist
 # this one gets errors still, dont know why
```

Comment:
No idea why this gives an error but perhaps you could inspect the different objects inside the function. Also, what is happening if the album / artist - name contains some weird characters? The German language has many of them (for instance ö,ä haha) or symbols, such as '# '*' - this might create some  strange errors even when they're in a string variable.